README.md

```diff
@@ -27,7 +32,7 @@ The source code for our documentation is [here](https://github.com/flexcompute-r
 
 Note that while this front end package is open source, to run simulations on Flexcompute servers requires an account with credits.
 You can sign up for an account [here](https://tidy3d.simulation.cloud/signup).
-After that, you can install the front end with the instructions below, or visit [this page](https://docs.flexcompute.com/projects/tidy3d/en/stable/quickstart.html) in our documentation for more details.
+After that, you can install the front end with the instructions below, or visit [this page](https://docs.flexcompute.com/projects/tidy3d/en/stable/install.html) in our documentation for more details.
```

Comment:
So there was no `install` page before if I recall, I think everything was consolidated into `quickstart`. I was thinking of structing the landing page as a proper `quickstart` and then renaming the old `quickstart -> install` since it seemed to me like the content made more sense that way?

Reply:
oh ok, my bad. I think we can try this.